### ISSUE TITLE
How can I add some new data to my model without training it from scratch?

### ISSUE THREAD

**Issue by ralekz** [ultralytics/ultralytics#22661]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I've reread your documentation and numerous forums, and I've asked questions on your website, but haven't found a solution. Please help.
For example, I need to accurately identify Toyota cars down to the model year. I trained the yolo11 detect model from scratch on a dataset of 400,000 labeled frames. The model works correctly and is satisfactory. However, every year Toyota adds new cars or updates the appearance of their vehicles.
Question: How can I correctly add new and restyled cars to the model I've already trained without retraining it from scratch? So that the model's excellent performance can be complemented by predictions for new cars.
Second question: How can I correctly retrain my model on a completely new dataset, but for, say, Nissan vehicles?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @ralekz, thank you for your interest in Ultralytics 🚀! This is an automated response to help get things moving. An Ultralytics engineer will also assist you here shortly.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question (like yours about extending YOLO11 for new or restyled vehicles and adapting to new brands), please share as much detail as possible so we can give targeted guidance:
-Your current `ultralytics` version, OS, CUDA, and PyTorch versions
-The exact YOLO11 model variant and last working training command/hyperparameters
-How your dataset/classes are defined now vs. what you plan to add or change
-Approximate size of the new data and a few example images
-Any relevant training logs/metrics and current model performance summaries
-Whether you plan to keep a single unified model or separate models per brand/category
-Any constraints (compute budget, training time, no-retrain-from-scratch requirement) 🧩

Also verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a **Python>=3.8** environment with **PyTorch>=1.8** to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including CUDA/CUDNN, Python and PyTorch preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by ralekz** [comment]
Clarifications:
1) YOLO11 detection
2) GeForce RTX 5070
3) Dataset of 360,000 cropped 640/640 frames, carefully labeled with boxes
4) 10% empty frames without labeling added
5) Toyota models are shown as examples; in reality, there are only 6 classes
6) Periodically, it is necessary to urgently add a new appearance for one or more classes without retraining the model from scratch.

**Comment by Y-T-G** [comment]
You can use your pretrained model as a base to train. But you can't avoid using the whole old dataset plus your new data. There's no easy way around it because of the way deep learning models work.
